src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: External services
+metaDescription: "This is an introduction to the external services feature"
+redirects:
+  - /docs/apm/apm-ui-pages/monitoring/external-services-page-view-web-cloud-network-data/
+---
+The external services feature allows you to look closely at the upstream and downstream activity of a single service. Those upstream or downstream external services may be your own services you instrumented, or they may refer to services you simply call in the course of a transaction. 
+
+The external services feature is no longer just for New Relic APM agent users. It has expanded to include OpenTelemetry and now uses the power of distributed tracing to give you insights into service activity. The classic external services feature is still available behind a UI toggle, but we recommend you try out the full power the expanded external services.  
+
+This feature provides a combination of maps and graphs to help you:
+
+* Find sources of latency and errors
+* Assess the impact of latency to upstream callers
+* Review transaction-level performance
+* Find traces that involve the transactions you're interested in
+
+To use the external services feature, you need a New Relic account with an installed APM agent with distributed tracing enabled. If you don't already have an account, you can sign up [here](https://newrelic.com/signup) for a free account.
```

Comment:
Style guide recommends be more specific than "here" for links. Also avoids clickbait impression. :-) https://docs.newrelic.com/docs/style-guide/formatting/hyperlinks . I would just change that to "If you don't already have one, [sign up](https://newrelic.com/signup) for a free account."

Reply:
@barbnewrelic, I changed this!